conduct.md

```diff
@@ -120,15 +120,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Openness. It is important to be vulnerable in your group so that they understand your strengths and weaknesses within coding. Being open will help your team members know what you are capable of doing so they do not assign you something that you would not understand. This could prevent confusion and reserve time. Being open can also allow your team members to help you with problems with your code that you did not catch or understand.
```

Comment:
All of these points contain great ideas! But, they don't conform to the format that the class agreed upon in [issue  #1](https://github.com/cmpsc-203-s22/code-of-conduct/issues/1). Specifically, they are not one sentence long and do not start with "Students...", as the class decided. Could they be reworded to follow this format? It would help the consistency throughout the conduct guide.